substrate/frame/election-provider-multi-phase/test-staking-e2e/src/lib.rs

```diff
@@ -322,24 +322,24 @@ fn automatic_unbonding_pools() {
 		assert_eq!(<Runtime as pallet_nomination_pools::Config>::MaxUnbonding::get(), 1);
 
 		// init state of pool members.
-		let init_free_balance_2 = Balances::free_balance(2);
-		let init_free_balance_3 = Balances::free_balance(3);
+		let init_stakeable_balance_2 = pallet_staking::asset::stakeable_balance::<Runtime>(&2);
```

Comment:
Maybe worth a `use` here to shorten these since they're used a few times. Likewise in other files with the new functions